i18n-msg-behavior.html

```diff
@@ -97,6 +101,13 @@
           value: {}
         },
 
+        /**
+         * i18n object, by default it contains all translations.
+         */
+        i18n: {
+          type: Object,
+          value: {}
```

Comment:
Yes and if an element instance does not want all translate properties it can set only the properties that it want.
You think it is a problem ?